src/main/scala/fly/play/jiraExceptionProcessor/ReportingSupervisorStrategy.scala

```diff
@@ -42,15 +35,25 @@ class ReportingStrategyWrapper(wrapped: SupervisorStrategy, comment: String) ext
 
   private def reportException(context: ActorContext, exception: Throwable) = {
     val configuration = Configuration(context.system.settings.config)
+
     val wsClient = {
       val environment = Environment.simple(mode = Mode.Prod)
-      val wsConfig = new WSConfigParser(configuration, environment).parse
-      val ningWsConfig = new NingWSClientConfigParser(wsConfig, configuration, environment).parse
-      NingWSClient(ningWsConfig)
+
+      val parser = new WSConfigParser(configuration, environment)
+      val config = new AhcWSClientConfig(wsClientConfig = parser.parse())
+      val builder = new AhcConfigBuilder(config)
+      val ahcConfig = builder.configure().build()
+
+      implicit val system = ActorSystem()
```

Comment:
It's good practice to give the actor system a name. Otherwise the name will become `default` which does not help with debugging.